### DISCUSSION TITLE
Cache Busting with Query Params

### DISCUSSION THREAD

**Discussion by tbvgl** [hyperjumptech/monika#854] [Q&A] [answered]
Is there a way to use dynamic query params for cache busting?

Example:

```
"probes": [
    {
      "id": "1",
      "name": "my_site",
      "requests": [
        {
          "url": "https://mysite.com?cachebust={{timestamp}}",
          "body": {},
          "timeout": 10000
        }
      ],
      "incidentThreshold": 5,
      "recoveryThreshold": 5,
      "alerts": []
    }
```

so the goal in the example above would be to always use the current timestamp as a value to check the uncached site.

**Comment by nicnocquee** [comment, marked as answer]
No way to do it currently, but that's a good idea! Will make an issue for that.

**Comment by nicnocquee** [comment]
@tbvgl we just released this feature in 1.12.0. Can you give it a try? the documentation can be found [here](https://monika.hyperjump.tech/guides/probes#fake-data).

**Reply by tbvgl** [reply]
It is working great @nicnocquee! I‘m building a dashboard for Monika. Thanks for implementing this. You guys rock.
